libs/core/algorithms/include/hpx/parallel/container_algorithms/transform.hpp

```diff
@@ -21,13 +23,17 @@ namespace hpx { namespace ranges {
     ///                     It describes the manner in which the execution
     ///                     of the algorithm may be parallelized and the manner
     ///                     in which it executes the invocations of \a f.
-    /// \tparam Rng         The type of the source range used (deduced).
-    ///                     The iterators extracted from this range type must
-    ///                     meet the requirements of an input iterator.
-    /// \tparam OutIter     The type of the iterator representing the
-    ///                     destination range (deduced).
+    /// \tparam FwdIter1    The type of the source iterators for the first
+    ///                     range used (deduced).
     ///                     This iterator type must meet the requirements of an
-    ///                     output iterator.
+    ///                     forward iterator.
+    /// \tparam Sent1       The type of the end source iterators used (deduced).
```

Comment:
Is Sent1 always an iterator? Could be value, no?

Reply:
Something like " The type of the end source **elements** used (deduced)." sounds better to you?

Reply:
nai

Reply:
Thought you copy-pasted from smwr. If it's from the non-ranges algos then we should come up with a different wording. Cause the ranges-algos have a different end type.

Reply:
It is a copy-paste, so maybe we should discuss about it today because I am not sure what the proper description is.

Reply:
True.

Reply:
Sentinels are a special type of 'end-iterator'. I think it is ok to use the term 'end source iterator' as in the above text.

Reply:
Alright, I'll leave it as it is then